client/src/components/CredentialsButton/CredentialsButton.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./CredentialsButton.css";
+import { PropTypes } from "prop-types";
+
+export const CredentialsButton = ({ data, onClick }) => {
+  return (
+    <button className="credentials" onClick={onClick}>
+      {" "}
```

Comment:
Adding in the spaces here seems rather unnecessary. If you wanted to add in some extra space, padding can be used instead

Reply:
Fixed: removed spaces